extensions/cornerstone/src/commandsModule.ts

```diff
@@ -65,6 +65,33 @@ const getLabelmapTools = ({ toolGroupService }) => {
   return labelmapTools;
 };
 
+const MARKER_TOOLS = ['MarkerInclude', 'MarkerExclude'];
+
+const getActiveLabelmapTools = ({ toolGroupService }) => {
```

Comment:
Can you call this
getPreviewTools
and then instead of having named tools, get the actual tool instance, and filter for .accept containing tools which are not disabled?  That way we can add more preview types in the future and have them accept/reject it.  
Alternatively, make the accept/reject functionality a general labelmap static instance method which can get called on any labelmap tool.

Reply:
Done